src/System.Transactions.Local/tests/AsyncTransactionScopeTests.cs

```diff
@@ -90,340 +90,349 @@ public AsyncTransactionScopeTests(ITestOutputHelper output)
         [InlineData(53)]
         [InlineData(54)]
 
-        public void AsyncTSTest(int variation)
+        public async Task AsyncTSTest(int variation)
         {
-            using (var listener = new TestEventListener(new Guid("8ac2d80a-1f1a-431b-ace4-bff8824aef0b"), System.Diagnostics.Tracing.EventLevel.Verbose))
+            await Task.Run(async delegate
             {
-                var events = new ConcurrentQueue<EventWrittenEventArgs>();
-
-                bool success = false;
-                try
+                var tcs = new TaskCompletionSource<bool>();
+                var ignored = Task.Delay(1).ContinueWith(_ =>
```

Comment:
What's the purpose of this tcs?  And, for that matter, of the delay?

Reply:
Task ignorance on my part. I was using some code in the tests for System.Threading.Tasks as an example. I will go back and re-examine.